src/app/leaderboard/LeaderboardTable.tsx

```diff
@@ -0,0 +1,119 @@
+"use client";
```

Comment:
the page itself is a server component, if we don't include this nextjs will throw an error because we can't have useState and similar client state things in a server component. If we make the page itself a client component using 'use client' then we can't use the filesystem stuff.